sdk/core/core-http/src/policies/deserializationPolicy.ts

```diff
@@ -75,10 +82,12 @@ export const DefaultDeserializationOptions: DeserializationOptions = {
 export class DeserializationPolicy extends BaseRequestPolicy {
   public readonly jsonContentTypes: string[];
   public readonly xmlContentTypes: string[];
+  public readonly xmlCharKey: string;
 
   constructor(
     nextPolicy: RequestPolicy,
     deserializationContentTypes: DeserializationContentTypes | undefined,
```

Comment:
is there a reason these are all ` | undefined` instead of making the parameters themselves optional with `?`?

Reply:
The last parameter is non-optional.  I could move both parameters to be after request policy options